src/main/plugins/org.dita.html5/xsl/hazard-d.xsl

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+This file is part of the DITA Open Toolkit project. 
+See the accompanying license.txt file for applicable licenses.
+-->
+<xsl:stylesheet version="2.0"
+     xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+     xmlns:ditamsg="http://dita-ot.sourceforge.net/ns/200704/ditamsg"
+     xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+     xmlns:xs="http://www.w3.org/2001/XMLSchema"
+     exclude-result-prefixes="ditamsg dita-ot xs">
+  
+  <xsl:variable name="inline-hazard-svg" as="element()">
+    <svg class="hazardsymbol" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg"
```

Comment:
Could we stick to some `resources` directory as a separate file and read with `document()`?

Reply:
:+1: 

Since we're using the same image for both HTML5 and XHTML - would you prefer to create a new `org.dita.base/resources/` directory and stick it there? I don't want HTML5 to depend on XHTML (or vice versa), so I think the other option is just to have a `resources/` directory for each plugin with a copy of the image. Only one copy seems better, but having two copies would keep the HTML5 / XHTML plugins more separate ... I'm probably fine either way.

Reply:
Currently I consider XHTML a legacy plugin and I don't really care about it. They have a lot of duplicated code, both having the image file is OK by me.